examples/python-package/subprojects/nanoarrow.wrap

```diff
@@ -15,13 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
-[wrap-file]
-directory = arrow-nanoarrow-apache-arrow-nanoarrow-0.5.0
-source_url = https://github.com/apache/arrow-nanoarrow/archive/refs/tags/apache-arrow-nanoarrow-0.5.0.tar.gz
-source_filename = apache-arrow-nanoarrow-0.5.0.tar.gz
-source_hash = 0ceeaa1fb005dbc89c8c7d1b39f2dba07344e40aa9d885ee25fb55b4d57e331a
-source_fallback_url = https://github.com/mesonbuild/wrapdb/releases/download/nanoarrow_0.5.0-1/apache-arrow-nanoarrow-0.5.0.tar.gz
-wrapdb_version = 0.5.0-1
+# Normally you should meson wrap install nanoarrow to populate this file; however
+# we use wrap-git to ensure this example is testing the latest version in our CI.
+[wrap-git]
+url = ../../..
```

Comment:
I think you need one more level of navigation upwards for this particular wrap file